frontend/e2e/app.spec.ts

```
@@ -0,0 +1,166 @@
+  const sendButton = page.getByRole("button", {name: "Send"});
+  await sendButton.click();
+
+  // Wait for error (alert or error message)
+  // Since alert, we can listen for dialog
+  page.on("dialog", async dialog => {
+    expect(dialog.message()).toContain("Could not perform request");
+    await dialog.accept();
+  });
+
+  // Verify error handling occurred
+  await new Promise(resolve => setTimeout(resolve, 2000));
```

Biome (2.1.2)

[error] This code will never be reached ...

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)